src/platyPS/platyPS.psm1

```diff
@@ -2763,13 +2765,28 @@ function ConvertPsObjectsToMamlModel
     $Inputs = @()
 
     $Help.inputTypes.inputType | ForEach-Object {
-        $InputObject = New-Object -TypeName Markdown.MAML.Model.MAML.MamlInputOutput
-        $InputObject.TypeName = $_.type.name
-        $InputObject.Description = $_.description |
-            DescriptionToPara |
-            AddLineBreaksForParagraphs
+        if ($_.description -eq $null -and $_.type.name -ne $null)
+        {
+            $inputtypes = $_.type.name.split("`n")
```

Comment:
> Also updated the parser to correctly parse input and output from the get-help call

This sounds very similar to #379
Can you show the original help? There could be missing `.INPUT` tags.

Reply:
Opening a new PR with the cherry-picked commits, but I'm a little confused by the comment.  I have in the description of the PR the pre and post output, and what the help looks like.  Do you need additional information?